### ISSUE TITLE
CTD after loading cold and dark aircraft at the gate or parking after 1.1.10.0 update

### ISSUE THREAD

**Issue by edueocara27** [kayjay1c6b/MSFS2024_AutoFPS-by-kayJay_1c6#3]
After today's update, I'm getting CTD with any aircraft I load cold and dark at the gate during the start up process with autoFPS open. And even if I open autoFPS only when I finish starting the aircraft and configuring everything, I still get CTD.

Before today's update, I did 3 flights with autoFPS and had no issues with any of them.

**Comment by edueocara27** [comment]
Trying to do a flight with the A320neo, I tried to open autoFPS only when I was in the air and I didn't get CTD right away. For a moment I thought I had found a workaround. AutoFPS worked without any problems during the flight, until the aircraft reached the top of the descent. The moment I selected the knob on the autopilot to start the descent, the simulator froze and then CTD.

I checked the event viewer and there are two error messages for each time I had CTD with autoFPS.

> Faulting application name: FlightSimulator2024.exe, version: 1.1.10.0, timestamp: 0x00000000
> Faulting module name: FlightSimulator2024.exe, version: 1.1.10.0, timestamp: 0x00000000
> Exception code: 0xc0000005
> Fault offset: 0x0000000001fe0ce3
> Faulting process ID: 0x1100
> Faulting application start time: 0x1DB451FB8741E8F
> Faulting application path: G:\SteamLibrary\steamapps\common\Limitless\FlightSimulator2024.exe
> Faulting module path: G:\SteamLibrary\steamapps\common\Limitless\FlightSimulator2024.exe
> Report ID: c66d8807-fb29-456b-b9a6-92276d2c6ab2
> Full name of the faulting package:
> Application ID for the faulting package:

And this other

> Autopilot.dll wil error was reported.
> HRESULT: 0x80070491
> File: onecoreuap\admin\moderndeployment\autopilot\dll\dllmain.cpp, line 128
> Message: NULL
> 

Hopefully this might give you some idea of ​​what's going wrong for me.

**Comment by edueocara27** [comment]
I did a clean reinstall of both MSFS 2024 and AutoFPS. Fresh install, zero mods in the community and the CTD problem persists. I also tried all the solutions in the FAQ and it didn't work.

I did some tests and this is the summary of what I found:

- Of all the aircraft I have tested so far, for some reason, the A330 (All Variants) and the Beluga XL are the only ones that are not affected by this problem, which is bizarre, since the A320neo and A321neo have this problem and both should share the same systems as the A330 since they are all made by iniBuilds.

- In some aircraft like the 737MAX and the 747-8, I have CTD as soon as I start pre-flight preparation, such as turn on the batteries and aligning the IRS. However, in most of them, I only have CTD after I import a flight plan into them, either from Simbrief or from Microsoft's flight planner. If I don't import any flight plan, they work normally with autoFPS.

- In the event viewer, the following error message appears every time I have CTD with autoFPS

> Autopilot.dll wil error was reported.
> HRESULT: 0x80070491
> File: onecoreuap\admin\moderndeployment\autopilot\dll\dllmain.cpp, line 128
> Message: NULL
>
